### PR TITLE
Homepage updates

### PR DESCRIPTION
**Pull request by danmitchell-**
### Changes proposed in this pull request
Remove content from home page and link to records view. Direct users back to the frontend pages.

### Guidance to review
View any register

# After
screen shot 2018-05-23 at 11 47 36

screen shot 2018-05-23 at 12 59 29

### CODE REVIEW

File: src/main/resources/templates/fragments/attribution.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <body>
   <th:block th:fragment="attribution">
     <div class="organisation">
-      <h2 class="heading-small">Managed by</h2>
```
Comment (michaelabenyohai) [inline src/main/resources/templates/fragments/attribution.html]: Can we delete this file as I'm not sure it's being used anymore?

File: src/main/resources/templates/fragments/organisation.html [hunk 2]
```diff
@@ -3,8 +3,7 @@
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <body>
   <th:block th:fragment="organisation(organisation, branding)">
-    <span th:if="${branding.present}" th:utext="${#strings.replace(branding.get().logoFormattedName,'\r\n','&lt;br/&gt;')}" class="organisation-logo" th:classappend="|organisation-logo-${branding.get().logoClassName} ${branding.get().colourClassName}|">Cabinet Office</span>
-    <span th:unless="${branding.present}" th:text="${organisation.name}" class="organisation-logo organisation-logo-no-identity">Cabinet Office</span>
+    <span th:text="${organisation.name}">Cabinet Office</span>
```
Comment (michaelabenyohai) [inline src/main/resources/templates/fragments/organisation.html]: Can we delete this file? It seems a bit overkill to have it as a fragment now as it's only being used in one place.

File: src/main/resources/templates/fragments/phase-banner.html [hunk 3]
```diff
@@ -7,7 +7,11 @@
       <div class="column-two-thirds">
         <div class="panel panel-border-wide phase">
           <span class="phase-banner">In progress</span>
-          <p>This data is in progress and it's not ready for use. <a href='https://registers.cloudapps.digital/support' target="_blank">Contact the team</a> to give us feedback.</p>
+          <p>
+            This data is in progress and it's not ready for use.
+            <br>
+            You can <a th:href="@{'https://www.registers.service.gov.uk/registers/' + ${registerId}}" target="_blank">preview the data and give us feedback</a>.
```
Comment (michaelabenyohai) [inline src/main/resources/templates/fragments/phase-banner.html]: This won't resolve if the phase is `discovery` or `test`.

File: src/main/resources/templates/home.html [hunk 4]
```diff
@@ -6,48 +6,42 @@
...
-              <li>
-                <a th:href="${homepageContent.registersIntroductionUrl}" target="_blank">About registers</a>
-              </li>
-            </ul>
-          </aside>
+          <h2 class="heading-small">Details</h2>
+          <ul class="register-metadata">
+            <li>
+              <span class="left">Name:</span>
+              <span class="right" th:utext="${friendlyRegisterId}"></span>
```
Comment (michaelabenyohai) [inline src/main/resources/templates/home.html]: This should be `${friendlyRegisterName}`

File: src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java [hunk 5]
```diff
@@ -57,23 +58,4 @@ public void homepageHasCopyrightInFooterRenderedAsMarkdown() throws Throwable {
...
-    public void homepageUsesRegisterIdForNameIfNameNotSpecified() {
-        Response response = register.getRequest(REGISTER_WITH_COPYRIGHT_FIELD, "/");
-        Document doc = Jsoup.parse(response.readEntity(String.class));
-
-        Elements registerNameHeading = doc.select("main .heading-large");
-        assertThat(registerNameHeading.first().text(), equalTo("Postcode register"));
-    }
-
-    @Test
-    public void homepageUsesRegisterNameIfExists() {
```
Comment (michaelabenyohai) [inline src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java]: These two tests are still relevant if we are correctly using `friendlyRegisterName` as above.

File: src/main/resources/templates/fragments/attribution.html (hunk 1 above)
Comment (danmitchell-) [reply src/main/resources/templates/fragments/attribution.html]: Yes, agreed. I need your help

File: src/main/resources/templates/fragments/organisation.html (hunk 2 above)
Comment (danmitchell-) [reply src/main/resources/templates/fragments/organisation.html]: Yes, agreed. I need your help

File: src/main/resources/templates/home.html (hunk 4 above)
Comment (michaelabenyohai) [reply src/main/resources/templates/home.html]: Just discussing this. We might use ID instead.

File: src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java (hunk 5 above)
Comment (michaelabenyohai) [reply src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java]: Ignore this as per discussion

File: src/main/resources/templates/entries.html [hunk 6]
```diff
@@ -6,6 +6,9 @@
     <header th:replace="fragments/global-header.html::global-header"></header>
 
     <main id="wrapper" role="main">
+      <th:block th:if="${register.phase == 'beta'}">
+        <div th:replace="fragments/register-intro.html::register-intro"></div>
```
Comment (gidsg) [inline src/main/resources/templates/entries.html]: Is it by design to only have this bit for beta registers?

Comment (danmitchell-) [reply src/main/resources/templates/entries.html]: Yes

File: src/main/resources/templates/fragments/register-intro.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+<html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+  <body>
+    <th:block th:fragment="register-intro">
+      <div class="registers-intro">
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <h1 class="heading-medium">Using the <th:block th:utext="${friendlyRegisterId}">register name</th:block> API</h1>
+            <p>We provide a secure API that is easy to integrate with so you can get the most up-to-date data from across government.</p>
+            <p>Create an API key to ensure that you always access the most recent version of all registers. The data is provided as JSON and CSV.</p>
+            <p><a th:href="@{'https://www.registers.service.gov.uk/registers/' + ${registerId}}" class="button">Get started</a></p>
```
Comment (gidsg) [inline src/main/resources/templates/fragments/register-intro.html]: Do we want to link directly e.g. to `https://www.registers.service.gov.uk/create-api-key?register=${registerId}`?

Comment (conordelahunty) [reply src/main/resources/templates/fragments/register-intro.html]: We want to link to the FE page for the register.

File: src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java [hunk 8]
```diff
@@ -4,6 +4,7 @@
 import org.jsoup.nodes.Document;
 import org.jsoup.select.Elements;
 import org.junit.Before;
+import org.junit.ClassRule;
```
Comment (gidsg) [inline src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java]: is this new import needed?

Comment (danmitchell-) [reply src/test/java/uk/gov/register/functional/HomePageFunctionalTest.java]: I'm not sure, @michaelabenyohai ?

**Pull request merged**